src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml

```diff
@@ -427,4 +432,12 @@ else
     </div>
     </div>
 }
-</form>
\ No newline at end of file
+</form>
+
+@functions {
+    // todo: move to sd shared
+    private static string AgeToString(int age)
+    {
+        return age == 127 ? "25+" : age.ToString();
+    }
```

Comment:
Question: where does 127 come from, and why do we treat it as 25+?

Reply:
This is a copy of another function. So I don't really have answers to that other than we do.